HomeWorkL3.py

```diff
@@ -0,0 +1,23 @@
+print("I love Python"*42)
+
+age_in_month = 39*12
+print(age_in_month)
+age_in_years = int(age_in_month / 12)
+print(age_in_years)
+name = 'Valentyna'
+my_age = f'Му name is {name}. I’m {age_in_years} years old'
+print(my_age)
+
+i = 1
+print(i == 'b')
+# print(i < 'ytr')
+print(i > 45)
+print(90 >= i)
+print(i != 'r')
+print(i <= -5)
+
+a = 2
+b = 5
+c = 6
+d = (c**a + b**a) * a**a + c*a

```

Comment:
The variable should contain a result of a concatenation of variables a, b and c. You can do it by transforming these variables to strings and adding them or using f-stings